.github/workflows/compute.yml

```diff
@@ -3,23 +3,17 @@ on:
   workflow_dispatch:
     inputs:
       eventName:
-        description: "Event ID"
+        description: "Event name"
         required: true
-      issueOwner:
-        description: "Organization that owns the repo"
-        required: true
-      issueRepository:
-        description: "Repository name"
-        required: true
-      issueNumber:
-        description: "Issue number"
-        required: true
-      collaborators:
-        description: "Repository collaborators"
+      event:
+        description: "Event payload"
         required: true
       installationId:
         description: "Installation ID"
         required: true
+      settings:
+        description: "Plugin settings"
+        required: true
```

Comment:
I like that the kernel is compiling the `ubiquibot-config.yml` for the plugins but what about character limits for transmitting data to GitHub Action inputs?

Our configs can get quite long. I'm wondering if it makes sense for plugins to get the config if they need it. @FernandVEYRIER is working on an npm module for this specific scenario.